### ISSUE TITLE
T070: Implement training loop with employment-specific loss functions

### ISSUE THREAD

**Issue by bmcgauley** [bmcgauley/is-160-project#74]
**Task ID**: T070
**Status**: Completed
**Suggested Assignee**: Brian
---
See [tasks.md](../specs/001/tasks.md) for full context and implementation details.
**Labels**: phase-3.5, training

**Comment by bmcgauley** [comment]
Task T070 completed. See commit history for implementation details.
